node/bin/src/prover_api/prover_server.rs

```diff
@@ -50,6 +44,12 @@ struct AvailableProofsPayload {
     available_proofs: Vec<String>,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+struct BatchDataPayload {
+    block_number: u64,
+    prover_input: String,
+}
```

Comment:
nit: there was this comment ` // base64‑encoded little‑endian u32 array` on `prover_input` that is gone now. Not a big deal, but worth retaining